internal/core/ports/types.go

```diff
@@ -252,3 +252,30 @@ type BuildData interface {
 	GetCommit() string
 	GetDate() string
 }
+
+type AddPriceFeedReq struct {
+	MarketBaseAsset  string
+	MarketQuoteAsset string
+	Source           string
+	Ticker           string
+}
+
+type UpdatePriceFeedReq struct {
+	ID     string
+	Source string
+	Ticker string
+}
+
+type PriceFeed struct {
+	ID               string
+	MarketBaseAsset  string
+	MarketQuoteAsset string
+	Source           string
+	Ticker           string
+	On               bool
+}
```

Comment:
As you can notice, there are no structs defined here, only interfaces in order to make it usable from both the app and the interface layers.
We may refactor this, but, as always, for now let's stick with the actual style and eventually refactor in a dedicated PR:
```suggestion
type PriceFeed struct {
	GetId() string
	GetMarket() Market
	GetSource() string
	GetTicker() string
	IsStarted() bool
}
```

Reply:
Done